client/hawc_client/base_client.py

```diff
@@ -0,0 +1,29 @@
+from io import StringIO
```

Comment:
@casey1173 quick comment here; you'll still need an `__init__.py` file b/c that's what makes a python package. You'll also need to keep the client in there that we typicallly use. It'll probably look something like...

```python

__all__ = ["HawcClient"]
__version__ = "2020.10"

from .somwhere import HawcClient 
```

That way all the APIs work like they previously did.